Geometry/HcalTowerAlgo/src/HcalFlexiHardcodeGeometryLoader.cc

```diff
@@ -86,7 +86,7 @@ std::vector<HcalFlexiHardcodeGeometryLoader::HBHOCellParameters> HcalFlexiHardco
 		  << "]: " << etabins[i].layer[k].first-1 << ":"
 		  << etabins[i].layer[k].second << std::endl;
 #endif
-	result.push_back (HcalFlexiHardcodeGeometryLoader::HBHOCellParameters(iring, depth, etabin.phis[j].first, etabin.phis[j].second, dphi, rmin, rmax, etabin.etaMin, etabin.etaMax));
+	result.emplace_back (HcalFlexiHardcodeGeometryLoader::HBHOCellParameters(iring, depth, etabin.phis[j].first, etabin.phis[j].second, dphi, rmin, rmax, etabin.etaMin, etabin.etaMax));
```

Comment:
should the constructor be removed here (and for the others in this file and the other loader)?